scripts/west_commands/completion/west-completion.bash

```diff
@@ -754,6 +760,10 @@ __comp_west_build()
 			__set_comp_west_boards
 			return
 			;;
+		--shield|-i)
```

Comment:
please update zsh and fish in addition.
https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/west_commands/completion/west-completion.fish
https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/west_commands/completion/west-completion.zsh

Reply:
I don't use either of those shells, sorry.

Reply:
a bit unfortunate, I use the `fish` shell so can take a look in future.

In the meantime, can you create a follow-up enhancement issue, so that at least we have some visibility that completion of shields are missing for zsh and fish ?